docs/framework/wcf/diagnostics/exceptions-reference/security-exceptions.md

```diff
@@ -3,150 +3,152 @@ title: "Security Exceptions"
-|UnableToLoadCertificateIdentity|Cannot load the X.509 certificate identity specified in the configuration.|  
-|UnexpectedEmptyElementExpectingClaim|The specified element from the specified  namespace is empty and does not specify a valid identity claim.|  
-|UnknownEncodingInBinarySecurityToken|Unrecognized encoding occurred while reading the binary security token.|  
-|UnsecuredMessageFaultReceived|An unsecured or incorrectly secured fault was received from the other party. See the inner FaultException for the fault code and detail.|  
-|UnsupportedPasswordType|The specified username token has an unsupported password type.|  
-|UnsupportedSecureConversationBootstrapProtectionRequirements|Cannot import the security policy. The protection requirements for the secure conversation bootstrap binding are not supported. Protection requirements for the secure conversation bootstrap must require both the request and the response to be signed and encrypted.|  
+
+This topic lists all security exceptions.
+
+## Exception List
+
+|Resource Code|Resource String|
+|-------------------|---------------------|
+|AnonymousLogonsAreNotAllowed|The service does not allow you to log on anonymously.|
+|AtLeastOneContractOperationRequestRequiresProtectionLevelNotSupportedByBinding|The request message must be protected. This is required by an operation of the specified contract. The protection must be provided by the specified binding.|
+|AtLeastOneContractOperationResponseRequiresProtectionLevelNotSupportedByBinding|The response message must be protected. This is required by an operation of the specified contract. The protection must be provided by the specified binding.|
+|AtMostOnePrimarySignatureInReceiveSecurityHeader|Only one primary signature is allowed in a security header.|
+|BadContextTokenFaultReason|The security context token expired or is not valid. The message was not processed.|
+|BadEncryptionState|The EncryptedData or EncryptedKey is in an invalid state for this operation.|
+|BasicHttpMessageSecurityRequiresCertificate|BasicHttp binding requires that BasicHttpBinding.Security.Message.ClientCredentialType be equivalent to the BasicHttpMessageCredentialType.Certificate credential type for secure messages. Select Transport or TransportWithMessageCredential security for UserName credentials.|
+|BasicTokenCannotBeWrittenWithoutEncryption|The basic token cannot be written without encryption.|
+|BindingDoesNotSupportProtectionForRst|The specified binding for the specified contract is configured with SecureConversation, but the authentication mode is not able to provide the request/reply-based integrity and confidentiality required for the negotiation.|
+|BindingDoesNotSupportWindowsIdentityForImpersonation|The specified contract operation requires Windows identity for automatic impersonation. A Windows identity that represents the caller is not provided by the specified binding for the specified contract.|
```

Comment:
should be "BindingDoesNotSupportWindowsIdenityForImpersonation" instead of "BindingDoesNotSupportWindowsIdentityForImpersonation"